tools/FlatFlowMigrationCli/Operations/RollbackOperation.cs

```diff
@@ -0,0 +1,79 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using FlatFlowMigrationCli.Options;
+using Microsoft.DotNet.ProductConstructionService.Client;
+using Microsoft.Extensions.Logging;
+using Tools.Common;
+
+namespace FlatFlowMigrationCli.Operations;
+
+internal class RollbackOperation : IOperation
+{
+    private readonly IProductConstructionServiceApi _pcsClient;
+    private readonly ISubscriptionMigrator _subscriptionMigrator;
+    private readonly ILogger<RollbackOperation> _logger;
+    private readonly RollbackOptions _options;
+
+    public RollbackOperation(
+        ILogger<RollbackOperation> logger,
+        IProductConstructionServiceApi pcsClient,
+        RollbackOptions options,
+        ISubscriptionMigrator subscriptionMigrator)
+    {
+        _logger = logger;
+        _pcsClient = pcsClient;
+        _options = options;
+        _subscriptionMigrator = subscriptionMigrator;
+    }
+
+    public async Task<int> RunAsync()
+    {
+        Console.Write("This is not a dry run, changes to subscriptions will be made. Continue (y/N)? ");
```

Comment:
I guess it's not important since we'll prob never use this tool after doing the migration, but we are repeating this code here. Could be a static method in the interface I guess?